pkg/osx/scripts/postinstall

```diff
@@ -34,10 +34,20 @@ if [ ! -f /etc/salt/minion ]; then
 fi
 
 ###############################################################################
+# Create symlink to salt-config.sh
+###############################################################################
+# echo "Symlink: Creating symlink for salt-config..." >> /tmp/postinstall.txt
+if [ ! -d "/usr/local/sbin" ]; then
+    mkdir /usr/local/sbin
```

Comment:
I thought `bin` was for binaries and `sbin` was for scripts. Since this is a script it should go here. I had to create the directory on my system because it's a brand new image.
The customer who requested this suggested the script should go in `/usr/local/sbin`